bin/heudiconv

```diff
@@ -935,6 +946,46 @@ def convert(items, symlink=True, converter=None,
     shutil.rmtree(tmpdir)
 
 
+def get_formatted_scans_key_row(item, outname_bids_files):
+    dcm_fn = item[-1][0]
+    mw = ds.wrapper_from_data(dcm.read_file(dcm_fn, stop_before_pixels=True))
+    # we need to store filenames and acquisition times
+    # parse date and time and get it into isoformat
+    date = mw.dcm_data.AcquisitionDate
+    time = mw.dcm_data.AcquisitionTime.split('.')[0]
+    td = time + date
+    acq_time = datetime.strptime(td, '%H%M%S%Y%m%d').isoformat()
+    # add random string
+    randstr = ''.join(map(chr, sample(k=8, population=range(33, 127))))
+    row = [acq_time, mw.dcm_data.PerformingPhysicianName, randstr]
+    return row
+
+
+def save_scans_key(items, outname_bids_files):
+    header = ['filename', 'acq_time', 'operator', 'randstr']
+    rows = []
+    for item, outname_bids_file in zip(items, outname_bids_files):
+        # get filenames
+        f_name = '/'.join(outname_bids_file.split('/')[-2:])
+        f_name = f_name.replace('json', 'nii.gz')
+        rows.append(
+            [f_name] + get_formatted_scans_key_row(item, outname_bids_file)
+        )
+    # where should we store it?
+    output_dir = dirname(dirname(outname_bids_file))
+    # get subject info
+    subj_pattern = re.compile('(sub-[a-zA-Z0-9]*)')
+    subj = subj_pattern.findall(f_name)
+    assert(len(subj) >= 1)
+    subj = subj[0]
+
+    # save
+    with open(pjoin(output_dir, '{0}_scans.tsv'.format(subj)), 'w') as csvfile:
```

Comment:
what if we are adding more data (another session) here.  So the file already exists with some fields... IMHO we should load it up, and add/update with new rows, otherwise we would loose data